blueocean-personalization/src/main/js/components/DashboardCards.jsx

```diff
@@ -94,7 +94,9 @@ export class DashboardCards extends Component {
           cards={pausedCards}
```

Comment:
https://gyazo.com/0718d4136a623ec3925a394139ebb09d

See above. You have specified a default value for `favoritesCardStack` but not for `pausedCardStack`